lib/matplotlib/tests/test_ticker.py

```diff
@@ -546,6 +547,21 @@ def test_use_offset(self, use_offset):
             tmp_form = mticker.ScalarFormatter()
             assert use_offset == tmp_form.get_useOffset()
 
+    def test_use_locale(self):
+        conv = locale.localeconv()
+        sep = conv['thousands_sep']
+        if not sep or conv['grouping'][-1:] in ([], [locale.CHAR_MAX]):
```

Comment:
Ugh, is there some way to do this *without* adding lines inside the test that aren't executed? On my todo list is to get us to the point where we're actually running 100% of the test code on CI. Could this somehow go in a `skipif` decorator?

Reply:
It'd be a very long `skipif`... Could we add `nocov` marker instead?

Reply:
That'd work for me.